neard/src/lib.rs

```diff
@@ -203,6 +203,11 @@ pub fn apply_store_migrations(path: &String, near_config: &NearConfig) {
         // version 19 => 20: fix execution outcome
         migrate_19_to_20(&path, &near_config);
     }
+    if db_version <= 20 {
+        info!(target: "near", "Migrate DB from version 20 to 21");
+        // version 20 => 21: delete genesis json hash
```

Comment:
can you add a bit more explanation as to why we need to delete genesis json hash?